README.md

```diff
@@ -1,11 +1,16 @@
 
 # curve25519-dalek  [![](https://img.shields.io/crates/v/curve25519-dalek.svg)](https://crates.io/crates/curve25519-dalek) [![](https://docs.rs/curve25519-dalek/badge.svg)](https://docs.rs/curve25519-dalek) [![](https://travis-ci.org/isislovecruft/curve25519-dalek.svg?branch=master)](https://travis-ci.org/isislovecruft/curve25519-dalek)
 
-**A low-level cryptographic library for point, group, field, and scalar
+<table>
```

Comment:
Not sure if you like the border or not, but you might consider `border="0"` (omg old skool)